src/app/zap-templates/zcl/data-model/chip/test-cluster.xml

```diff
@@ -155,5 +155,18 @@ limitations under the License.
       <arg name="returnValue" type="INT8U"/>
     </command>
 
+    <command source="server" code="0x02" name="TestAddArrayofStructArguments" optional="false">
```

Comment:
Can we add both complex client-side and server-side command requests and responses respectively?

Perhaps, just two commands sequences in total:

`TestSimpleArgumentRequest` --> `TestStructArrayArgumentResponse`.
`TestStructArrayArgumentRequest --> `TestSimpleArgumentResponse`.

Reply:
it has been moved in https://github.com/project-chip/connectedhomeip/pull/10171